integration-tests/hibernate-orm-panache/src/main/java/io/quarkus/it/panache/resources/BookResource.java

```diff
@@ -19,6 +22,11 @@
     @Path("/{name}/{author}")
     public List<Book> addAndListAll(@PathParam("name") String name, @PathParam("author") String author) {
         bookDao.persist(new Book(name, author));
-        return bookDao.listAll();
+        List<Book> books = bookDao.listAll();
+        // This is done to invoke the hopefully replaced field access with the getter access
+        books.forEach(book -> {
+            book.name.hashCode();
```

Comment:
I don't understand how this tests for using the getters?

In ORM+Panache we have a lots of tests for that, which involve calling real getters and making sure they were called. Why don't you copy that?

Reply:
because this call (`getField`) replaced by `invokeVirtual` which is a public method where the field is private and so avoids the `IllegalAccessException`.

Reply:
I'm sorry this doesn't parse.

Reply:
No worries.  I found the accessor tests on the hibernate side and I'm recreating them for mongodb.  this will all go away then.